tests/test_parmparse.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+import amrex.space3d as amr
+
+
+def test_parmparse():
+    dopml = False
+    ncell = 10
+    dt = 0.0
+    # Since ParmParse is done in IO Processors, we need amr.initialize first
+    amr.initialize([])
```

Comment:
Contrary to plain scripts, these `pytest` tests initialize AMReX before calling `test_...`. This is done here:
https://github.com/AMReX-Codes/pyamrex/blob/39f7f33142f119c4c40c4db865086ae527bf7a3c/tests/conftest.py#L22-L38

So we can remove this.
```suggestion

```